m::AddError(const char* message, const char*)
{
  this->Errors.push_back(message);
};